rstcheck.py

```diff
@@ -981,6 +981,7 @@ def main():
         except (IOError, UnicodeError) as exception:
             output_message(exception)
             status = 1
+        pool.close()
```

Comment:
What about putting this in the finally block of the try block above?

Reply:
@stsewd is that the option you would prefer?

If you're happy with the idea of the change, how about using pool as a context manager?

So changing line 958 from `pool = multi...` to `with multi... as pool`

Reply:
Resolving as moved to the finally block as per suggestion from @stsewd